packages/subnav/partials/MenuItemsDefault/style.module.css

```diff
@@ -0,0 +1,96 @@
+.root {
+  display: flex;
+  padding: 0 32px;
+  margin: 0 auto;
+  align-items: center;
+  list-style: none;
+
+  &.alignRight {
+    margin-right: 0;
+    padding-right: 16px;
+  }
+}
+
+.listItem {
+  position: relative;
+  white-space: nowrap;
+  margin: 0;
+  padding: 0;
+}
+
+.navLink {
+  composes: g-type-body-small-strong from global;
+  position: relative;
+  padding-left: 16px;
+  padding-right: 16px;
+  line-height: 2.5rem;
+  display: flex;
+  align-items: center;
+  white-space: nowrap;
+}
+
+.submenuItem {
+  composes: g-type-body-small-strong from global;
+  color: var(--black);
+}
+
+/*
+Note: duplicated in MenuItemsDefault
+*/
```

Comment:
This seems like a good indication to abstract the pattern into a component, at least internally within `Subnav` 💭

Reply:
💖  Great call - ✅ abstracted out into a mostly-just-styles `NavItemText` component.